tests/Query/BuilderTest.php

```diff
@@ -247,6 +321,34 @@ public static function provideQueryBuilderToMql(): iterable
                 ]),
         ];
 
+        /** @see DatabaseQueryBuilderTest::testLimitsAndOffsets() */
+        yield 'offset limit' => [
+            ['find' => [[], ['skip' => 5, 'limit' => 10]]],
+            fn (Builder $builder) => $builder->offset(5)->limit(10),
+        ];
+
+        yield 'offset 0 limit 0' => [
+            ['find' => [[], []]],
+            fn (Builder $builder) => $builder->offset(0)->limit(0),
```

Comment:
Offhand, do you know if zero is handled exactly the same as "null (reset)"?

If so, I'd be in favor of the following changes:

 * Rename this test to "offset limit zero (unset)"
 * Create a new test below named "offset limit null (unset)", based on this test. The point of these tests would be to demonstrate that zero and null values both result in the options being unspecified.
 * Create a new test here named "offset limit zero (reset)", based on the "offset limit null (reset)" test below. Like that, it would demonstrate that zero and null both reset the options back to an unset state

Reply:
This comment might have been missed.

I'll defer to you if you'd like to make the suggested test changes. Otherwise, feel free to resolve.

Reply:
Test updated. Thanks for the reminder.